### ISSUE TITLE
Improving equity between different disabilities

### ISSUE THREAD

**Issue by jspellman** [w3c/wcag3#230]
Part of the Silver Research project of 2017-18 were interviews on Conformance with a small number of [thought leaders from different aspects of accessibility industry](https://docs.google.com/document/d/1ZD8T1XgcZJAlV7VketWAx8f3Doad05jU5lpVf7wuIW0/edit#heading=h.9i2nf5iv1l3n).  They were promised anonymity to encourage free comments.  They were given a standard list of questions.  They reached interesting and informative [conclusions ](https://docs.google.com/document/d/1ZD8T1XgcZJAlV7VketWAx8f3Doad05jU5lpVf7wuIW0/edit#heading=h.y4qfie9jmt3v)and directions for the future.  Note that this was a small group of participants that were selected for their innovative and thoughtful thinking on broad accessibility issues.  

One issue raised by the two US-based lawyers interviewed was the question of how success criteria were assigned to different levels and that "Testable" was included in the criteria of how the success criteria were assigned.  One lawyer with Civil Rights experience was concerned that if a judge questioned the inclusion of testability, that there was a substantial risk that the WCAG regulations would be struck down.  The lawyers interviewed strongly recommended that user needs be the basis of evaluating accessibility and testability not be included in any assignment of level of priority or severity.  

A later concern that was raised during the Design Sprint from a US accessibility lawyer is that the current assignment of success criteria to the levels is not balanced by disability type.  For example, the Deaf, deaf, and hearing impaired have only one A success criterion (1.2.2 captioning), 1 AA (1.2.4 Captions (Live) and all others are AAA.  Any solution in WCAG3 should strive to treat all disabilities equally.  One suggestion was to require a minimum level in each disability category.  

I am raising this issue, because as we move into AG discussions of conformance, I do not want the research that was done to be forgotten. There are innovative solutions in the Silver Research and Silver Design Sprint that should not be lost because AG members naturally want to build on what is familiar.  There is an opportunity for a transformative approach to conformance that can address important problems in accessibility.  That should not be lost.  

In summary: 

- Read the [Silver Research](https://www.w3.org/WAI/GL/task-forces/silver/wiki/Silver_Research_Archive), especially the [Interviews on Conformance Conclusions](https://docs.google.com/document/d/1ZD8T1XgcZJAlV7VketWAx8f3Doad05jU5lpVf7wuIW0/).  
- Remove testability from any evaluation of severity or priority.  Focus on user needs. This will ensure long-term viability of WCAG in regulatory use.  
- Ensure that attention is paid to equity between disability categories.  All disabilities have essential user needs.

**Comment by hidde** [comment]
Thanks for this important reminder. 

The [latest working draft](https://www.w3.org/TR/wcag-3.0/) contains a new Conformance section following conformance discussions in AGWG and its conformance subgroup. In general, like earlier drafts, this working draft is based on outcomes informed by user needs. We also explicitly list functional performance statements. 

Closing this issue as the conformance discussions have happened (though they may continue at a later stage). Feedback on the new conformance and reporting sections is welcomed!

**Comment by alastc** [comment]
Just to note how (and what bits) the conclusions have been incorporated:

> One participant recommended decoupling testability and conformance so that conformance requirements do not restrict what guidance can be included in Silver. 

The guidance (requirements/assertions/recommended practices) go beyond conformance, and reporting enables for organizations to report improvements below and above conformance.

> The lawyers interviewed strongly recommended that user needs be the basis of evaluating accessibility and testability not be included in any assignment of level of priority or severity.

Testability isn't part of leveling. Although it is part of whether a requirement is included, it doesn't define what level it goes to (as there aren't levels at the conformance level). There are also recommended practices and assertions that are not considered as testable as requirements.

> Any solution in WCAG3 should strive to treat all disabilities equally. One suggestion was to require a minimum level in each disability category.

The group is tracking which requirements benefit which functional needs, but the aim is to include everything we can.

> recommended looking at the US and Canada LEEDS (Leadership in Energy and Environmental Design) standards for green building certification as a model for Silver conformance and certification.  They proposed a “bronze” level certification for meeting minimum Silver conformance (the equivalent of WCAG 2.0 AA), then a point system where organizations could increase their score up to a “platinum” level by steadily improving the usability for people with disabilities.

That is essentially the approach taken, with some terminology differences. As an "output" standard (that assesses the output of design/development), the term "conformance" has a very specific meaning, but we have layered on reporting to get to WCAG Bronze/Silver/Gold levels.

> International, non-native English speaking assistive technology vendors do not understand how to implement specifications correctly. 

This is ongoing with the accessibility supported work.

> Both advocates commented that user needs should have a higher priority than regulation. 

For WCAG, being used in regulation is an important part of getting in front of the most people possible. In the guideline level guidance there will be a listing of user-needs so that the relationship is clearer.
